website/views/shared/tasks/edit/tags.jade

```diff
@@ -1,5 +1,5 @@
 fieldset.option-group(ng-if='!$state.includes("options.social.challenges")')
   p.option-title.mega(ng-class='{active: task._tags}', ng-click='task._tags = !task._tags', tooltip=env.t('expandCollapse'))=env.t('tags')
   label.checkbox(ng-repeat='tag in user.tags', ng-if='task._tags')
-    input(type='checkbox', ng-model='task.tags[tag.id]')
+    input(type='checkbox', ng-model='task.tags[tag.id]', ng-checked="task.tags.indexOf(tag.id) !== -1")
```

Comment:
I'll be able to remove `ng-model='task.tags[tag.id]` once I the userService is ported.